### PR TITLE
[Communication]: Update missing  release notes for communication

### PR DESCRIPTION
**Pull request by JianpingChen**

### CODE REVIEW

File: releases/2021-02/js.md [hunk 1]
```diff
@@ -42,6 +48,46 @@ If you have a bug or feature request for one of the libraries, please post an is
 
 [pattern]: # (### ${PackageFriendlyName} ${PackageVersion} [Changelog]${ChangelogUrl}`n${HighlightsBody}`n)
 
+### Azure Communication Common 1.0.0-beta.5 [ChangeLog](https://github.com/Azure/azure-sdk-for-js/blob/@azure/communication-common_1.0.0-beta.5/sdk/communication/communication-common/CHANGELOG.md#100-beta5-2021-02-09)
```
Comment (RezaJooyandeh) [inline releases/2021-02/js.md:51]: ### Azure Communication Administration will be deprecated

- Identity client is moved to new package Azure Communication Identity.
- Phone number administration will be moved into a new package Azure Communication Phone Numbers.

File: releases/2021-02/dotnet.md [hunk 2]
```diff
@@ -126,6 +126,56 @@ If you have a bug or feature request for one of the libraries, please [file an i
 - Renamed `SearchIndexingBufferedSenderOptions<T>.MaxRetryDelay` to `SearchIndexingBufferedSenderOptions<T>.MaxThrottlingDelay`.
 - Renamed `SearchIndexingBufferedSenderOptions<T>.RetryDelay` to `SearchIndexingBufferedSenderOptions<T>.ThrottlingDelay`.
 - Removed the helper method `SearchClient.CreateIndexingBufferedSender<T>()`. Instead, callers are expected to use the public constructor of `SearchIndexingBufferedSender<T>`.
+### Azure Communication Administration will be deprecated
+
+- Identity client is moved to new package Azure Communication Identity.
+- Phone number adminstration will be moved into a new package Azure Communication Phone Numbers.
```
Comment (RezaJooyandeh) [inline releases/2021-02/dotnet.md:132]: adminstration -> administration

File: releases/2021-02/ios.md [hunk 3]
```diff
@@ -37,13 +38,15 @@ To add the Azure SDK for iOS to your application, follow the example in [Importi
 Open your project's `Package.swift` file and add a new package dependency to your project's `dependencies` section, specifying the clone URL of the repository and the version specifier you wish to use:
 
 ```swift
-// Insert dependencies here
+...
```
Comment (RezaJooyandeh) [inline releases/2021-02/ios.md:41]: It seems that we have not update the highlight section for ios.

File: releases/2021-02/python.md [hunk 4]
```diff
@@ -51,6 +55,35 @@ the identity of the connection endpoint.
 - `Style` is renamed to `TextStyle`.
 - Client property `api_version` is no longer exposed. Pass keyword argument `api_version` into the client to select the API version.
 
+### Azure Communication Identity 1.0.0b4 [ChangeLog](https://github.com/Azure/azure-sdk-for-python/blob/azure-communication-identity_1.0.0b4/sdk/communication/azure-communication-identity/CHANGELOG.md#100b4-2021-02-09)
```
Comment (RezaJooyandeh) [inline releases/2021-02/python.md:58]: ### Azure Communication Administration will be deprecated

- Identity client is moved to new package Azure Communication Identity.
- Phone number administration will be moved into a new package Azure Communication Phone Numbers.

File: releases/2021-02/ios.md (hunk 3 above)
Comment (JianpingChen) [reply releases/2021-02/ios.md:41]: This is no changelog.md for iOS. I had nothing to copy-n-past from. ;-)

Comment (JianpingChen) [reply releases/2021-02/ios.md:41]: Actually, Victor is handling iOS in his PR already. I'll revert my changes.

https://github.com/Azure/azure-sdk/pull/2401

**Review by RezaJooyandeh — APPROVED** [review]

**Pull request merged**